.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Please add `/node_modules` directory in `.gitignore ` file. Because, it can be added by accident while setting up Linters.